docs/interfaces/auth-service.txt

```diff
@@ -1,104 +1,134 @@
 Interface Token {
-    payload: {
-        // The lifetime of the Token. The date in unix seconds of the expiration.
-        expiresAt: date,
-        // The corresponding userId of the requesting client.
-        userId: number,
-        // The id of the current session.
-        sessionId: string
-    },
-    signature: string
+    // The corresponding userId of the requesting client.
+    userId: number,
+    // The id of the current session.
+    sessionId: string,
+    // The lifetime of the Token. The date in unix seconds of the expiration.
+    exp: number,
+    // Date of creation of a token.
+    iat: number
 }
 
 Interface Cookie {
-    // The id for the session corresponding to the client, who has signed in.
+    // The id for the session corresponding to the user's client.
     sessionId: string,
-    // A signature created from the server.
-    signature: string
+    // The lifetime of a cookie. Date of expiration in unix seconds.
+    exp: number,
+    // Date of creation of a token in unix seconds.
+    iat: number
+}
+
+// The properties of this interface have to be passed as HTTP-headers in a request. 
+Interface Ticket {
+    authentication: string,
+    cookies: 'refreshId=string'
+}
+
+/**
+* Describes an http-response, which is sent back to any requesting service.
+*/
+Interface Response <T> {
+    // Optional headers, which are set in an http-response
+    HttpHeaders: {
+        // Authentication is passed, if a new access-token is returned.
+        authentication?: string,
+        // A cookie containing 'refreshId=(Cookie as string)' is passed, if a user signs in.
+        set-cookie?: refreshId=string
```

Comment:
Suggestion:

```
cookies: {
    [name: string]: string
}
```
This has the advantage, that you can control the cookie name, or even support multiple cookies. This has to be adopted everywhere.

Please add to the docs:
- the lifetime of all cookies:
- other flags, like HTTPOnly